### PR TITLE
chore: update CODEOWNERS for Datastore

### PR DESCRIPTION
**Pull request by kolea2**

### CODE REVIEW

File: apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj [hunk 1]
```diff
@@ -11,7 +11,6 @@
     <PackageReference Include="Google.Apis.CloudResourceManager.v1" Version="[1.57.0.2677, 2.0.0.0)" />
     <ProjectReference Include="..\..\..\tools\Google.Cloud.ClientTesting\Google.Cloud.ClientTesting.csproj" />
     <ProjectReference Include="..\Google.Cloud.PubSub.V1\Google.Cloud.PubSub.V1.csproj" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
```
Comment (jskeet) [inline apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj:14]: Is this deliberate? My guess is not.

Comment (kolea2) [reply apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj:14]: No, this looks like it was done by owlbot? https://github.com/googleapis/google-cloud-dotnet/pull/9877/commits/4080ef35c9ad1d1cdf22fa0eed1cda2a3321a19f

Comment (jskeet) [reply apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj:14]: Ah yes - I understand now. Will fix tomorrow and merge this then.

Comment (kolea2) [reply apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj:14]: No rush, thanks!

**Review by jskeet — APPROVED** [review]

**Pull request merged**